.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+node_modules
```

Comment:
`node_modules` -> `node_modules/`
This will ignore all directories named `node_modules` in folders and subfolders